tsconfig.json

```diff
@@ -7,139 +7,14 @@
         "preserveConstEnums": true,
         "declaration": true,
         "sourceMap": true,
-        "inlineSources": true,
```

Comment:
Please don't remove this. I've explained it a few times now, this is needed for debugging when you install from NPM.